docs/types/reference-types.rst

```diff
@@ -132,11 +132,11 @@ a reference to it.
 
 .. index:: ! string, ! bytes
 
-.. _strings:
+.. _string:
```

Comment:
I think this should not be changed.

Reply:
@FeiyangTan If this is changed the references need to be changed too.